targettest/heaptest.c

```diff
@@ -6,7 +6,7 @@
 
 
 
-static unsigned char* V[256];
+static char* V[256];
```

Comment:
This program uses binary bytes, not text characters.  Those type declarations should be `unsigned char`, not `char`.  "heaptest.c" shouldn't be changed.

Reply:
OK, I was wrong.  The program does need to be fixed.  But, it's the `Alloc()` function's declaration that's wrong.

Reply:
No clue what's wrong with Alloc(), I'm afraid.

Reply:
`Alloc()` wants to return the value of `P`.  Therefore, its return type must match the type of `P`.  The first warning says that it doesn't match.  `P` is correct; therefore, it's the return type that needs fixing.

Reply:
Am confused. Am not getting any warnings.

Reply:
That's because you changed some things.  You need to go back to the original repository version:
```
git restore heaptest.c
```
Then, you'll see five warnings when you compile "heaptest.c".  The first warning is the important one; the others are only consequences of the first one.  It points to https://github.com/cc65/cc65/blob/6ac4aa4e2076189ad5b5f3a9d33e1c5910d4e37c/targettest/heaptest.c#L31
It says that the `return` and the `P` have different pointer types.  The type of the `return` is declared by https://github.com/cc65/cc65/blob/6ac4aa4e2076189ad5b5f3a9d33e1c5910d4e37c/targettest/heaptest.c#L13
The bug is that an `unsigned` is missing from that line.

Reply:
So I guess I'd have been easier to fix oneself than to write a ticket. I'm out.